core/templates/dev/head/pages/exploration_player/ExplorationFooterDirective.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for showing author/share footer
+ * in exploration player.
+ */
+
+oppia.directive('explorationFooter', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      twitterText: '@'
+    },
+    templateUrl: 'components/explorationFooter',
+    controller: [
+      '$scope', '$http', '$log', 'explorationContextService',
+      'UrlInterpolationService', 'ExplorationSummaryBackendApiService',
+      function(
+          $scope, $http, $log, explorationContextService,
+          UrlInterpolationService, ExplorationSummaryBackendApiService) {
+        $scope.explorationId = explorationContextService.getExplorationId();
+
+        $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+
+        $scope.contributorNames = [];
+
+        $scope.getTwitterText = function() {
+          return $scope.twitterText;
+        };
+
+        ExplorationSummaryBackendApiService
+          .loadPublicAndPrivateExplorationSummaries(
+                [$scope.explorationId]).then(function(summaries) {
+              var summaryBackendObject = null;
+              if (summaries.length > 0) {
+                $scope.contributorNames = (
+                  Object.keys(
+                    summaries[0].human_readable_contributors_summary).sort(
+                      function(contributorUsername1, contributorUsername2) {
+                        var commitsOfContributor1 = dict[
```

Comment:
@joshuacano -- There's reference to a "dict" here, but that dict is not defined, and this results in a frontend error. I noticed this when testing the case of 2 contributors. This needs to be fixed...

To simulate:
1. Log in as A, then create and publish an exploration.
2. In a separate browser or incognito window, sign up a new user, B.
3. A then adds B as a collaborator in the Settings tab of A's exploration.
4. B goes to A's exploration and edits it.
5. Go to /explore/{{exploration_id}} and you should see the error.

Thanks!